templates/partials/summary/list-summary.html

```diff
@@ -1,38 +1,43 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
-{% if list.list_items | length > 0 %}
-
-  {% if list.editable %}
-    {% set headers = ["Name of person", "Action"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person",
-      _("Change"),
-      _("Change details for {item_name}"),
-      _("Remove"),
-      _("Remove {item_name}")
-    ) %}
-  {% else %}
-    {% set headers = ["Name of person"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person"
-    ) %}
-  {% endif %}
-
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "headers": headers,
-            "rows": rows,
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
-
+{% if list.editable %}
+  {% set headers = ["Name of person", "Action"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person",
+    _("Change"),
+    _("Change details for {item_name}"),
+    _("Remove"),
+    _("Remove {item_name}")
+  ) %}
+{% else %}
+  {% set headers = ["Name of person"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person"
+  ) %}
 {% endif %}
+
+{{ onsSummary({
+  "withinQuestion": true,
+  "summaries": [
+    {
+      "groups": [
+        {
+          "groupTitle": list_title,
+          "headers": headers,
+          "rows": rows,
+          "summaryLink": {
+            "url": add_link,
+            "text": add_link_text,
+            "attributes": {
+              "data-qa": "add-item-link"
+            }
+          },
```

Comment:
As `summaryLink` is always added, when there is no `add_link` or `add_link_text` the following html is incorrectly added to the html
```html
<div class="summary__add-link u-bt"><a data-qa="add-item-link" href></a></div>
```

Reply:
I've added `add_link` conditional now, sets `summaryLink` as an attribute of group config if the link exists. Thanks for help!